docs/reference/urllib3.util.rst

```diff
@@ -4,14 +4,20 @@ Utilities
 Useful methods for working with :mod:`http.client`, completely decoupled from
 code specific to **urllib3**.
 
-At the very core, just like its predecessors, :mod:`urllib3` is built on top of
+At the very core, just like its predecessors, urllib3 is built on top of
 :mod:`http.client` -- the lowest level HTTP library included in the Python
 standard library.
 
-To aid the limited functionality of the :mod:`http.client` module, :mod:`urllib3`
+To aid the limited functionality of the :mod:`http.client` module, urllib3
 provides various helper methods which are used with the higher level components
 but can also be used independently.
 
+.. autoclass:: urllib3.util.timeout._TYPE_DEFAULT
```

Comment:
`_TYPE_*` types shouldn't be documented, they're private APIs.

Reply:
Then we will have a lot of warnings like:

`WARNING: py:class reference target not found: urllib3.util.timeout._TYPE_DEFAULT`